oneanddone/users/tests/test_api.py

```diff
@@ -96,14 +97,18 @@ def test_delete_user(self):
         self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token.key)
         test_user = UserFactory.create()
         user_uri = self.uri + test_user.email + '/'
+        get_response = User.objects.get(username=test_user.username)
 
         # Make a DELETE request
         delete_response = self.client.delete(user_uri)
         self.assert_response_status(delete_response, status.HTTP_204_NO_CONTENT)
 
         # Verify that the user has been deleted
-        get_response = self.client.get(user_uri)
-        self.assert_response_status(get_response, status.HTTP_404_NOT_FOUND)
+        try:
+            get_response = User.objects.get(username=test_user.username)
+        except:
```

Comment:
Using a catch-all `except` is not good practice as this will catch any exceptions, not just the one we are expecting. Instead, catch only the expected exception (which I believe is `DoesNotExist`), and if that exception is not thrown then you can fail the test. You won't need the `eq_()` below, because all you want to know is that the `DoesNotExist` exception was thrown.

You could also, alternatively, use `filter` instead of `get` and make sure that the queryset is empty. That would avoid having to catch an exception at all, so might be more readable.